compiler/rustc_mir_build/src/build/expr/as_operand.rs

```diff
@@ -94,32 +94,35 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
     ///
     /// Like `as_local_call_operand`, except that the argument will
     /// not be valid once `scope` ends.
+    ///
+    /// If `last_operand` is set to true, we can skip the intermediate temporary for constants.
```

Comment:
Is this left over from some earlier version of your changes? Only other mention of something like this is in the commit message as far as I can tell.